docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc

```diff
@@ -0,0 +1,85 @@
+= How to use the OpenShift integrated container registry
+
+The OpenShift integrated registry of each {product} zone is exposed outside the cluster at `registry.<zone>.appuio.cloud`.
+
+== Prerequisites
+
+To pull from the registry, your account needs the `registry-viewer` role.
+To push to the registry, your account needs the `registry-editor` role.
+These permissions are generally namespace-scoped.
+Both of these permissions are part of the `edit` and `admin` project roles.
+The `registry-viewer` role is also part of the `view` project role.
+
+See the https://docs.openshift.com/container-platform/latest/registry/accessing-the-registry.html#prerequisites[OpenShift docs] for details on how to configure those roles for your user.
+
+== Accessing the registry with your {product} user
+
+You can access the registry with your {product} credentials with the following steps:
+
+. Login to the desired zone and create docker credentials for the zone registry
++
+[source,bash]
+----
+zone=cloudscale-lpg-0 <1>
```

Comment:
```suggestion
zone=cloudscale-lpg-2 <1>
```

Just to be future proove.

Reply:
Do we care more about future-proofing the docs or about providing docs that actually work when copy-pasting? If we want future-proof docs, and don't care about copy-pastability we can directly do something like 

```suggestion
zone=ZONE-ID <1>
```

And adjust the callout to state that users need to replace `ZONE-ID` with an existing zone.